db/structure.sql

```diff
@@ -2,8 +2,8 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 9.6.9
--- Dumped by pg_dump version 9.6.9
+-- Dumped from database version 9.6.16
```

Comment:
We have to use that version what our server use.

And if we change version, that would be better if we do that in a separated commit. But now it modified the structure only a little, so it isn't a big problem.

Reply:
According to [docker-compose.yml](https://github.com/kir-dev/pek-next/blob/master/docker-compose.yml#L9) our server uses the latest version of Postgresql 9.6.x at the time of creation of the image. (Which is in prod 9.6.11). Although it's a good idea not to mix the versions, but I think for patch version we can be permissive.